<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
background-color:qlineargradient(spread:pad, x1:0.49, y1:0, x2:0.502, y2:1, stop:0 rgba(61, 55, 52, 255), stop:0.664145 rgba(47, 42, 39, 255))
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.49, y1:0, x2:0.502, y2:1, stop:0 rgba(61, 55, 52, 255), stop:0.664145 rgba(47, 42, 39, 255))
}</string>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1031</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#background{
border-radius:10px;
border-width:3px;
border-color: rgb(120, 106, 93);
border-style: solid;
background-color: rgb(43, 39, 36);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>139</y>
      <width>1031</width>
      <height>531</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scroll_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1031</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Skip to previous update (Hotkey: Left Arrow)</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#previous{
border-top-left-radius:5px;
border-top-right-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt; Prev</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Skip to next update (Hotkey: Right Arrow)</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#next{
border-top-left-radius:5px;
border-top-right-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>Next &gt;&gt;</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="update_n">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#update_n{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="update_d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#update_d{
border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Alphabetical">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Sort Alphabetically</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Alphabetical{
border-top-left-radius:5px;
border-top-right-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReleaseOrder">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Sort by Release Order</string>
    </property>
    <property name="toolTipDuration">
     <number>2500</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ReleaseOrder{
border-top-left-radius:5px;
border-top-right-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>calendar_icon.png</normaloff>calendar_icon.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>90</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:5px;
border-top-right-radius:5px;
border-color: rgb(251, 236, 203);
background-color: rgb(251, 236, 203);
color: black</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_text">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
border-color: rgb(120, 106, 93);
background-color: rgb(120, 106, 93);</string>
    </property>
    <property name="text">
     <string>âœ•</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
